services/resource-usage-tracker/tests/unit/with_dbs/conftest.py

```diff
@@ -105,6 +113,35 @@ def _creator(**overrides) -> dict[str, Any]:
     return _creator
 
 
+@pytest.fixture
+def random_resource_tracker_credit_transactions(
+    faker: Faker,
+) -> Callable[..., dict[str, Any]]:
```

Comment:
I was suggesting something like the below. This will run it in all 3 flavours.

```suggestion
@pytest.fixture(prams=["BILLED", "PENDING", "NOT_BILLED"])
def transaction_status(request: purest.FixtureRequest)-> str:
    return request.param

@pytest.fixture
def random_resource_tracker_credit_transactions(
    faker: Faker, transaction_status: str
) -> Callable[..., dict[str, Any]]:
```

Reply:
aa got it thanks!